cmd/troubleshoot/cli/run.go

```diff
@@ -346,6 +259,162 @@ the %s Admin Console to begin analysis.`
 	return nil
 }
 
+// loadClusterSpecs loads the support bundle and redactor specs from the cluster
+// based on troubleshoot.io/kind=support-bundle label selector. We search for secrets
```

Comment:
According to https://github.com/replicatedhq/kURL/pull/4247, it should be  troubleshoot.sh, right?

Reply:
not until we've come to a conclusion on that path

Reply:
awesome, the PR is all good now.